navigation/navigation.css

```diff
@@ -0,0 +1,91 @@
+@import url(/style.css);
+@import url(/badge/badge.css);
+
```

Comment:
most of your code was in the initial commit. Try to change the commit message before pushing origin.